### PR TITLE
Update pytest selenium

### PR DESCRIPTION
**Pull request by SeanRayR**
image

### CODE REVIEW

**Review by HeffZilla — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -27,3 +34,29 @@
...
+
+```pip install pytest```
+
+Install pytest and pytest-selenium
+
+```pip install pytest pytest-selenium```
+
+Install addition dependencies
+
+```pip install webdriver-manager```
```
Comment (wgunn-quality) [inline README.md]: I don't think we need to list these out separately since they are/should be defined in the requirements file.

File: README.md [hunk 2]
```diff
@@ -10,6 +10,13 @@
 
 ### Test Explorer settings.json file
 
+##Setup
+1) Install Python 3.9
+https://www.python.org/downloads/ (this also installs the python package-management system - pip)
+2) Set Python as the interpreter in your IDE
```
Comment (wgunn-quality) [inline README.md]: I think this title should be "2) Set up virtual environment"
At least based on the link underneath.

File: test/conftest.py
Comment (wgunn-quality) [inline test/conftest.py:1]: I think webdriver_manager is old style. We can simply do this:
```python
import pytest
from selenium import webdriver


@pytest.fixture
def chrome_browser():
    options = webdriver.ChromeOptions()
    options.add_argument("--headless")  # Run Chrome in headless mode
    service = webdriver.ChromeService()
    driver = webdriver.Chrome(options=options, service=service)
    yield driver
    driver.quit()
    ```

**Review by wgunn-quality — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (SeanRayR) [reply README.md]: updated

File: README.md (hunk 2 above)
Comment (SeanRayR) [reply README.md]: updated

File: test/conftest.py
Comment (SeanRayR) [reply test/conftest.py:1]: also update this

**Review by wgunn-quality — APPROVED** [review]

**Pull request merged**